server/libs/orcid.js

```diff
@@ -3,7 +3,6 @@ import xmldoc from 'xmldoc'
 import config from '../config'
```

Comment:
It doesn't look like you changed anything in this file, try to avoid adding files that don't have logical changes. If you do want to update whitespace, it should be in a commit saying so.